### ISSUE TITLE
Trouble with cycle 2  shuffle: Offset Stack 

### ISSUE THREAD

**Issue by donaldvallejo** [malsup/cycle2#580]
this is my html. im not sure what wrong? am i missing a link or is there something that i have to type in.

<!doctypehtml>
<html>
<head>
    <title>Slide Show</title>

</head>
<body>

<div class="cycle-slideshow" 
    data-cycle-fx=shuffle
    data-cycle-timeout=3000
    data-cycle-shuffle-top="25"
    data-cycle-shuffle-left="-20"
    data-cycle-reverse=true
    data-cycle-hide-non-active=false
    >
    <img src="http://malsup.github.io/images/p1.jpg" data-cycle-slide-css='{ "top": "-10px",  "left": "10px" }'>
    <img src="http://malsup.github.io/images/p2.jpg" data-cycle-slide-css='{ "top": "20px",   "left": "20px" }'>
    <img src="http://malsup.github.io/images/p3.jpg" data-cycle-slide-css='{ "top": "10px",   "left": "-10px" }'>
    <img src="http://malsup.github.io/images/p4.jpg" data-cycle-slide-css='{ "top": "-20px",  "left": "-20px" }'>
</div>

</body>

<script src="http://ajax.googleapis.com/ajax/libs/jquery/1/jquery.min.js"></script>

<script src="http://malsup.github.io/jquery.cycle2.js"></script>

<script src="http://malsup.github.io/jquery.cycle2.shuffle.js"></script>

<script src="jquery.easing.1.3.js"></script>

<script src="http://malsup.github.io/jquery.cycle2.shuffle.js"></script>

</html>
